vertx-web/src/test/java/io/vertx/ext/web/RouterTest.java

```diff
@@ -1735,7 +1735,7 @@ public void testLocaleWithCountry() throws Exception {
       rc.response().end();
     });
 
-    testRequest(HttpMethod.GET, "/foo", req -> req.putHeader("Accept-Language", "en-gb;q=0.8, en;q=0.7, da_DK;q=0.9"), 200, "OK", null);
```

Comment:
I know it is not part of the spec but i think some httpclients (that rely on the JDK Locale will use _ instead) that was the reason we add it there. I'm not sure if we should be strict here or not...